src/nwb2bids/bids_models/_bids_session_metadata.py

```diff
@@ -28,8 +28,8 @@ class BidsSessionMetadata(BaseMetadataContainerModel):
         description="Timing data and metadata regarding events that occur during this experiment.", default=None
     )
     probe_table: ProbeTable | None = None
-    channel_table: ChannelTable | None = None
     electrode_table: ElectrodeTable | None = None
+    channel_table: ChannelTable | None = None
```

Comment:
I adjusted the ordering in this file to more naturally flow with the cross-dependency